newm-server/src/main/kotlin/io/newm/server/database/migration/V54__MarketplaceUpdates.kt

```diff
@@ -0,0 +1,25 @@
+package io.newm.server.database.migration
+
+import org.flywaydb.core.api.migration.BaseJavaMigration
+import org.flywaydb.core.api.migration.Context
+import org.jetbrains.exposed.sql.transactions.transaction
+
+@Suppress("unused")
+class V54__MarketplaceUpdates : BaseJavaMigration() {
+    override fun migrate(context: Context?) {
+        transaction {
+            exec(
+                """
+                CREATE TABLE IF NOT EXISTS marketplace_pending_orders (
```

Comment:
Since marketplace is much higher user count than the studio, we should really add indexing to it based on the queries we need to do. Create a jira ticket for adding indexes to marketplace before public launch.

Reply:
I'll create a separate task